PiBridgeMaster.c

```diff
@@ -873,6 +881,7 @@ int PiBridgeMaster_Run(void)
 			my_rt_mutex_lock(&piDev_g.lockPI);
 			pI1->drv = pI2->drv;
 			pI2->usr = pI1->usr;
```

Comment:
Now this assignment is not needed anymore, right?

Reply:
Also please explain in a comment why we do the memcopy here with the length given in i16uFBS_OutputLength